prometheus-to-sd/main.go

```diff
@@ -113,7 +113,18 @@ func main() {
 		glog.Error(http.ListenAndServe(fmt.Sprintf(":%d", *debugPort), nil))
 	}()
 
-	client := oauth2.NewClient(context.Background(), google.ComputeTokenSource(""))
+	var client *http.Client
+
+	if *config.ManualProject != "" {
```

Comment:
Let's be consistent about this condition (earlier you use *manualCluster != "" || *manualInstance != "" || *ManualProject != ""). I think we should just check the *manualProject != "" (just as here)
- If not set, assume running on GCE, but allow to overwrite all other parameters individually (e.g. if someone uses their own weird tooling to set up kubernetes cluster and have their own node names and cluster name that is not picked up by Metadata Server)
- If set, require all other flags / default to empty strings.

Reply:
okay, I can do that.

Reply:
okay, done. I think this is what you meant.